python.md

```diff
@@ -0,0 +1,216 @@
+# Python Guidelines
+
+There are many excellent Python style guides out there, usually based on [PEP
+8](https://www.python.org/dev/peps/pep-0008/) -- the mother of all Python style
+guides. Following the principle of reuse, we recommend an existing guides that
+we find suitable, and only add a few refinements and accentuation to it. In
+short:
+
+- Follow the recommendations of this document.
+- Follow the recommendations of the [Google Python Style
+  Guide](https://google.github.io/styleguide/pyguide.html) for everything that
+  is not covered in this document.
+- Follow local customs when working on existing code.
+- Knowing [PEP 8](https://www.python.org/dev/peps/pep-0008/) doesn't hurt.
+- Strive for readability and keep junior programmers in mind.
+- Maintainers and linters have the authority, which you may challenge if you
+  have good reason.
+- Add a suitable linter configuration to your project if it does not have one.
+  You can take inspiration from well-established Python lab-projects such as
+  [in-toto](https://github.com/in-toto/in-toto) and
+  [TUF](https://github.com/theupdateframework/tuf).
+- If you are familiar with auto-formatters such as `yapf`, `black`, `autopep8`,
+  etc. feel free to share a config file that adheres to this guideline
+  document. :)
+
+
+### Indentation and Line Continuation
+*[cf. Google Python Style Guide [Line
+length](https://google.github.io/styleguide/pyguide.html#32-line-length) and
+[Indentation](https://google.github.io/styleguide/pyguide.html#34-indentation)]*
+
+Use **4 spaces** per indentation level for new projects, but favor consistency
+in existing projects (most of them use 2 spaces).
+
+Favor **aligned indentation** over hanging indentation for line continuation
+and resolve potential problems like so:
+
+- **Problem:** Long first lines push continued lines to the right edge:
+
+  ```python
+  # NO:
+  import package.has.many.sub.packages.or_long_package_names.foo
```

Comment:
This import style is contrary to style recommended by the Google guide. Is this example necessary?

Reply:
AFAICS, the Google guide permits this one as well (see ["verbose" vs. "common" import style](https://google.github.io/styleguide/pyguide.html#233-decision)), so I thought it might be good to advise against it here.

Reply:
oh, so it does. Ugh. Then yes please let's advise against.